docs/en/start/config.md

```diff
@@ -0,0 +1,108 @@
+---
+sidebar_position: 5
+---
+
+# Intro to config file
```

Comment:
I do not think we should put `config.md` in the start section. This will make our structure not clear, because both local, docker, and k8s are the way to bring ST up, but config is for all kind of deployment. I personally prefer to add a new parent sidebar named `how to` or `concept` to put some "usage" or "common" docs like that. WDYT?

Reply:
You are right, maybe add a new parent sidebar will be more suitable.